src/TRE-API/Controllers/VaultCredentialsController.cs

```diff
@@ -0,0 +1,229 @@
+﻿using BL.Services.Contract;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using Swashbuckle.AspNetCore.Annotations;
+using System.ComponentModel.DataAnnotations;
+
+namespace TRE_API.Controllers
+{
+    [Route("api/v1/vaultvredentials")]
```

Comment:
```suggestion
    [Route("api/v1/vaultcredentials")]
```

Typo.

Although not sure if that's a desirable base route per below routing comments...?

for versioning not sure if we want to manually use `v1` in the routes this way, that feels like a whole design conversation (that we should have and make a decision about how to take forward consistently!)

There are configurable versioning libraries available which might work better (or might not)

Reply:
I was trying to follow the epic [TREFX-55](https://ukserp.atlassian.net/browse/TREFX-55) API Review notes to have some standards as Hazel mention we can decide and implement